src/components/flowchart/draw.js

```diff
@@ -98,6 +109,7 @@ export const drawNodes = function(changed) {
     return;
   }
 
+  const updateNodes = this.el.nodes;
```

Comment:
Why introduce this variable instead of just using `this.el.nodes`? Just for clarity/consistency/explicitness?

Reply:
Yeah exactly this is for all of those things as it's not obvious unless you know d3 well.